mappings/InventoryID-Bibtex.tsv

```diff
@@ -3432,401 +3432,395 @@ InventoryID	BibtexKey	Source
 2625	103697	ea
 2626	testen1997ossetic	ea
 2626	phoible2283	ea
-2627	308862	ea
```

Comment:
should I be worried that certain inventory IDs (like `2627` here) no longer have bib keys associated with them?

Reply:
Yes! Good catch. I was a bit confused because the entry is in the bibtex file:

@article{308862,
	Author = {Hiram Ring},
	Bibtexkey = {ring_jowai-pnar2011},
	Date-Added = {2017-07-09 10:13:10 +0000},
	Date-Modified = {2017-07-09 10:13:10 +0000},
	Fn = {eurasia\ring_pnar2011.pdf},
	Hhtype = {phonology},
	Inlg = {English [eng]},
	Journal = {Mon-Khmer Studies: A Journal of Southeast Asian Linguistics and Languages},
	Macro_Area = {Eurasia},
	Pages = {133-175},
	Src = {hh},
	Srctrickle = {hh#phon:Ring:Pnar},
	Title = {A phonetic description and phonemic analysis of Jowai-Pnar},
	Volume = {40},
	Year = {2011}}

but somehow (or in other words I screwed up) two rows which got truncated in the `InventoryID-Bibtex.tsv` file. Sorry! Update pushed @drammock

Reply:
OK.  One other question.  In the old version of the file, there were several inventory IDs that had 2 entries (i.e., appeared on 2 sequential lines with 2 different bib keys).  Now all of them have only 1 bib key.  Is that intentional?  The inventory IDs in question are 2666, 2692, 2698, 2752, 2778, 2779, 2781, 2931

Reply:
Some history here: Erich Round provided the inventories data, with his initial Glottocode assignments, a few years back:

https://github.com/bambooforest/phoible-scripts/tree/master/ER/data/raw

He had no bibtex at the time, so I went through Glottolog, by hand, and grabbed bibtex entries and named them by their Glottolog IDs for each inventory given his text references, e.g.:

https://glottolog.org/resource/reference/id/308862

Since then, Round has updated the inventories in terms of languages (e.g. some doculects got merged, others updated), but he also created his own bibtex entries for each inventory.

I decided to add in his bibtex file with ours and update our InventoryID-Bibtex.tsv from his own mapping from:

https://github.com/bambooforest/phoible-scripts/blob/master/ER/data/raw2019/inventory_biblio.txt

to:

https://github.com/bambooforest/phoible-scripts/blob/master/ER/data/raw2019/Australian_phonemes_for_PHOIBLE_20190118.tsv

because I believe he will maintain this resource more accurately than I can in the future.

For our index, I updated the bibtex entries to his 2019 inventories file:

https://github.com/bambooforest/phoible-scripts/blob/master/ER/data/formatted2019/ER_InventoryID-Bibtex.tsv

But do note that I didn't remove all the "old" bibtex entries by hand. This sounds like a lot of work that I can automate later, but right now I don't think it matter that we have more bibtex than entries.